wolverine.py

```diff
@@ -25,7 +33,41 @@ def run_script(script_name, script_args):
     return result.decode("utf-8"), 0
 
 
-def send_error_to_gpt(file_path, args, error_message, model):
+def send_error_to_gpt(file_path, args, error_message, model=DEFAULT_MODEL):
+    def json_validated_response(model, messages):
```

Comment:
nice idea, I saw 3.5-turbo especially had issues with this. Can you extract `json_validated_response` to be a standalone function? Seems a bit long to be an inner function

Reply:
sure!